From: Ramesh Iyyar <rameshi1@in.ibm.com>
Date: Tue, 16 Jun 2020 10:28:06 -0500
Subject: [PATCH] data/p10: Added backward support to pdbg compatible property

The op-build ci will use released version of openbmc image and this image
have pdbg library which is used p9 compatible property so, when using
p10 targets compatible property value in generated device tree with that
library, ipling is failing at 3.22 with following trace
"No functional MC unit target found on master chip" at sbe side.
So, added p9 targets compatible property value to support previous released
version of bmc image.

To use updated pdbg version in bmc, bmc will expect respective device tree
and the device tree is one of partition in pnor image and this image will
build in op-build so, there is cyclic dependency between bmc and pnor image
to release. To avoid cyclic dependency in image added backward support.

Once bmc image released with updated pdbg version to support
p10 targets compatible property value then this patch should reverted.

Signed-off-by: Ramesh Iyyar <rameshi1@in.ibm.com>
---
 data/p10/pdbg_compatible_propMapping.lsv | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/data/p10/pdbg_compatible_propMapping.lsv b/data/p10/pdbg_compatible_propMapping.lsv
index f4325c6..72cf0b0 100644
--- a/data/p10/pdbg_compatible_propMapping.lsv
+++ b/data/p10/pdbg_compatible_propMapping.lsv
@@ -1,17 +1,17 @@
 # Targets pdbg mapping info
 # Format : MRWTargetType:PdbgCompatiblePropValue
-unit-eq-power10:ibm,power10-eq
-unit-fc-power10:ibm,power10-fc
-unit-core-power10:ibm,power10-core
-unit-perv-power10:ibm,power10-chiplet
-unit-pauc-power10:ibm,power10-pauc
-unit-pec-power10:ibm,power10-pec
-unit-phb-power10:ibm,power10-phb
-unit-nmmu-power10:ibm,power10-nmmu
-unit-iohs-power10:ibm,power10-iohs
-unit-pau-power10:ibm,power10-pau
-unit-mc-power10:ibm,power10-mc
-unit-mi-power10:ibm,power10-mi
+unit-eq-power10:"ibm,power10-eq", "ibm,power9-eq"
+unit-fc-power10:"ibm,power10-fc", "ibm,power9-fc"
+unit-core-power10:"ibm,power10-core", "ibm,power9-core"
+unit-perv-power10:"ibm,power10-chiplet", "ibm,power9-chiplet"
+unit-pauc-power10:"ibm,power10-pauc", "ibm,power9-pauc"
+unit-pec-power10:"ibm,power10-pec", "ibm,power9-pec"
+unit-phb-power10:"ibm,power10-phb", "ibm,power9-phb"
+unit-nmmu-power10:"ibm,power10-nmmu", "ibm,power9-nmmu"
+unit-iohs-power10:"ibm,power10-iohs", "ibm,power9-iohs"
+unit-pau-power10:"ibm,power10-pau", "ibm,power9-pau"
+unit-mc-power10:"ibm,power10-mc", "ibm,power9-mc"
+unit-mi-power10:"ibm,power10-mi", "ibm,power9-mi"
 chip-processor-power10:"ibm,power-proc", "ibm,power10-proc", "ibm,processor"
 unit-mcc-power10:ibm,power10-mcc
 unit-omic-power10:ibm,power10-omic
-- 
1.8.3.1

